### PR TITLE
add schemapack@1.4.2 w/ npm auto-update

### LINKED ISSUE #10549: [Request] Add schemapack
**Library name:** schemapack
**Git repository url:** https://github.com/phretaddin/schemapack
**npm package url(optional):** https://www.npmjs.com/package/schemapack
**License(s):** MIT
**Official homepage:** 
**Wanna say something? Leave message here:**

"The fastest and smallest JavaScript object serialization library. Efficiently encode your objects in to compact byte buffers and then decode them back in to objects on the receiver. Integrates very well with WebSockets."

schemapack is typically used in both client and server.  I think it would be very convenient to have a copy on cdnjs for client use!  Thank you for running this fantastic service.

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/41693715-request-add-schemapack?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by sajochiu**
close #10549, cc @phretaddin

Pull request for issue: #10549
Related issue(s): # #
@x09326 please help me review this PR, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/phretaddin/schemapack
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/schemapack
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:7
   - Count of stars:294
   - Count of forks:16
 * NPM download stats (optional):
   - Downloads in the last day:10
   - Downloads in the last week:55
   - Downloads in the last month:338

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/schemapack/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "buffer",
+    "arraybuffer",
+    "net"
+  ],
+  "author": "phretaddin",
+  "license": "MIT",
+  "npmName": "schemapack",
+  "npmFileMap": [
+    {
+      "basePath": "build",
```
Comment (x09326) [inline ajax/libs/schemapack/package.json]: @sajochiu 
There is no `build` folder and `schemapack.min.js` in npm package.
Please modify the filemap and manually minify the main file.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by clairetsai818** [comment]
I'm working on adding this lib #11314

**Comment by PeterDaveHello** [comment]
Close this one due to no response on https://github.com/phretaddin/schemapack/issues/24, we can reopen in future.

**Pull request closed without merging**